data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -166,6 +169,12 @@ do
     scp -o PreferredAuthentications=publickey -o StrictHostKeyChecking=false -o UserKnownHostsFile=/dev/null -r -q "core@[${master}]:/tmp/artifacts-${GATHER_ID}/*" "${ARTIFACTS}/control-plane/${master}/"
 done
 
+cluster_bootstrap_gather
```

Comment:
Just noticed that the gathering of cluster resources has not been removed from earlier in this file. This call to `cluster_bootstrap_gather` should be moved up to replace the current resource gathering.